startup.py

```diff
@@ -0,0 +1,49 @@
+import argparse
+import subprocess
+
+parser = argparse.ArgumentParser(description="Runs all of the ingest stuff")
+parser.add_argument("data", help="File path to data")
+parser.add_argument("type", help="type of data being ingested")
+parser.add_argument("table", help="Name of table")
+parser.add_argument("layer_name", help="Name of the layer")
+args = parser.parse_args()
+
+
+subprocess.call("./sbt 'project ingest' assembly", shell=True)
+# os.system("./sbt < stuff.txt")
+
+jar = "ingest/target/scala-2.10/gt-admin-ingest-assembly-0.1.0-SNAPSHOT.jar"
+data = "file:" + args.data
+table = args.table
+
+data_type = "AccumuloIngestCommand"
+if(args.type == "temporal"):
+    data_type = "NexIngest"
+
+layer = args.layer_name
+crs = "EPSG:3857"
+pyramid = "true"
+clobber = "true"
+
+subprocess.call("zip -d" + jar + "META-INF/ECLIPSEF.RSA", shell=True)
```

Comment:
One thing that would be good to do with these python `subprocess` calls is to let `subprocess` handle putting the command together (escaping, etc.) by passing a list instead. This is the suggested way via the [python docs](https://docs.python.org/2/library/subprocess.html#frequently-used-arguments), makes refactoring a little easier, and keeps things platform independent.

Might also want to use `subprocess.check_call` since that will [stop execution](https://docs.python.org/2/library/subprocess.html#subprocess.check_call) if one of these commands fails.